scaladoc/src/dotty/tools/scaladoc/renderers/MemberRenderer.scala

```diff
@@ -173,16 +173,19 @@ class MemberRenderer(signatureRenderer: SignatureRenderer)(using DocContext) ext
       )
     )
 
-  private case class MGroup(header: AppliedTag, members: Seq[Member])
+  private case class MGroup(header: AppliedTag, members: Seq[Member], groupName: String)
 
   private def actualGroup(name: String, members: Seq[Member | MGroup]): Seq[AppliedTag] =
     if members.isEmpty then Nil else
     div(cls := "documentableList")(
       h3(cls:="groupHeader")(name),
-      members.map {
+      members.sortBy {
```

Comment:
Members are sort by poistion in the source files/files. Three is a way to sort members by name using buttons.